peribolos/knative.yaml

```diff
@@ -500,7 +501,7 @@ orgs:
             description: Knative Steering Committee
             maintainers:
             - csantanapr
-            - itsmurugappan
+            - puerco
```

Comment:
@salaboy  I think these lists need to be sorted (I tried to push a commit to you r branch but I can't)